sdk/template/template/package.json

```
@@ -42,6 +42,7 @@
     "clean": "rimraf --glob dist dist-* temp types *.tgz *.log",
     "execute:samples": "dev-tool samples run samples-dev",
     "extract-api": "tshy && dev-tool run extract-api",
+    "extract-snippets": "tshy && dev-tool run update-snippets",
```

nitpick: to match the command name?
```suggestion
    "update-snippets": "tshy && dev-tool run update-snippets",
```

---

Updated